library/src/main/java/com/android/mms/transaction/RetryScheduler.java

```diff
@@ -346,13 +346,13 @@ public static void setRetryAlarm(Context context) {
                     Intent service = new Intent(TransactionService.ACTION_ONALARM,
                                         null, context, TransactionService.class);
 
-                    // Workaround for using PendingIntent.FLAG_MUTABLE until compileSdkVersion is updated to 31.
+                    // Workaround for using PendingIntent.FLAG_IMMUTABLE until compileSdkVersion is updated to 31.
                     // Actual value from:
                     // https://android.googlesource.com/platform/frameworks/base.git/+/android-13.0.0_r18/core/java/android/app/PendingIntent.java#262
```

Comment:
ソースの参照先も変更しておいた方が良さそう

```suggestion
                    // https://android.googlesource.com/platform/frameworks/base.git/+/android-13.0.0_r18/core/java/android/app/PendingIntent.java#247
```